controllers/openstackcluster_controller.go

```diff
@@ -119,6 +119,16 @@ func (r *OpenStackClusterReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		return reconcile.Result{}, err
 	}
 
+	if openStackCluster.Spec.Bastion != nil && openStackCluster.Spec.Bastion.Enabled {
+		if openStackCluster.Status.Bastion == nil {
+			*openStackCluster.Status.Bastion = infrav1.BastionStatus{}
```

Comment:
I'm trying to use a pointer here to see if the status is actually written.
I'm open for suggestions.